recon_surf/N4_bias_correct.py

```diff
@@ -278,30 +276,30 @@ def normalize_wm_aseg(
         target_wm: float = 110.,
         target_bg: float = 3.
 ) -> sitk.Image:
-    """Normalize WM image [MISSING].
+    """
+    Normalize WM image [MISSING].
```

Comment:
MISSING -> so the white matter has a mean intensity of `target_wm` and the backgroud has intensity `target_bg`